tests/wpt/mozilla/tests/css/inline_block_percentage_margin.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
```

Comment:
nit: No need for extra `<html>`, `<head>` and `<body>` tags, feel free to drop them.